eng/ci/public-build.yml

```diff
@@ -49,6 +52,11 @@ extends:
         compiled:
           enabled: true
         runSourceLanguagesInSourceAnalysis: true
+      autobaseline:
+        enableForGitHub: true
```

Comment:
What will happen from enabling sdl autobaseline for PRs?

Reply:
https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/1es-pipeline-templates/features/sdlanalysis/autobaselining